### DISCUSSION TITLE
Per-switch VLAN Group?

### DISCUSSION THREAD

**Discussion by novaclark** [netbox-community/netbox#11717] [Q&A] [answered]
I'm setting up a greenfield netbox deployment in an organisation where multiple switches share a common vlan id, but the vlan is not trunked between switches. Each switch runs a DHCP relay, so that there is one DHCP address pool that is unique to each switch.

Thus I'd like to create a separate VLAN for each Switch in netbox, all sharing the same vlan id, but associated to a different Prefix. Some searching seems to confirm my hunch, that VLAN Groups are the way to do this, but it appears VLAN Groups are associated to Sites, Locations, and other categories that are common between my Switches. The problem then is that I have multiple switches in a common rack. If I create a VLAN Group and a Location for each Switch, then I can't place multiple Switches into the same Rack because netbox doesn't allow me to assign a single Rack to multiple Locations.

I think maybe I could use a different VRF for each switch. I know this isn't a correct usage of VRF, but I don't know that it matters in our organisation. I'm also not sure if it solves the problem, but I may try it.

Am I doing it wrong? Is there another / better way to go about this?

**Comment by jsenecal** [comment, marked as answer]
You can make a VLAN Group that has no association or Scope. You could then create one per switch.

**Reply by novaclark** [reply]
Simple. Thank you.
